tests/models/biogpt/test_modeling_biogpt.py

```diff
@@ -367,9 +367,9 @@ def test_model_from_pretrained(self):
             model = BioGptModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
-    # Copied from tests.models.opt.test_modeling_opt.OPTModelTest with OPT->BioGpt
+    # Adapted from tests.models.opt.test_modeling_opt.OPTModelTest with OPT->BioGpt
     def test_biogpt_sequence_classification_model(self):
-        config, input_dict = self.model_tester.prepare_config_and_inputs()
+        config, input_dict = self.model_tester.prepare_config_and_inputs_for_common()
```

Comment:
If the only difference with OPT is the `prepare_config_and_inputs`, a feature of copied from is that you can do the following: 
```python 
# Copied from tests.models.opt.test_modeling_opt.OPTModelTest with OPT->BioGpt, prepare_config_and_inputs-> prepare_config_and_inputs_for_common
```
(Just a FYI!)

Reply:
Thanks! I was not aware of this feature. I will make the necessary changes.